README.md

```diff
@@ -35,15 +35,15 @@ HTML comments are not preserved.
 But, perhaps you'd like to display sanitized HTML immediately in the browser for preview. Or ask the browser to do the sanitization work on every page load. You can if you want to!
 
 * Clone repository and install via npm
-* Run npm install and :
+* Run npm install and:
 
 ```bash
 npm install sanitize-html # yarn install sanitize-html
 ```
 
-The primary change in the 2.x version of sanitize-html is that it no longer includes a build that is ready for browser use. Developers are expected to include sanitize-html in their project builds (e.g., webpack) as they would any other dependency. So while sanitize-html is no longer ready to link to directly in HTML, developers can now more easily process it according to their needs.
+The primary change in the 2.x version of `sanitize-html` is that it no longer includes a build that is ready for browser use. Developers are expected to include `sanitize-html` in their project builds (e.g., webpack) as they would any other dependency. So while `sanitize-html` is no longer ready to link to directly in HTML, developers can now more easily process it according to their needs.
```

Comment:
These are the name of the package, which is "sanitize-html" (for better or worse). So I wouldn't format them as code.

Reply:
Alright, I'm going to rename them all to sanitize-html, w/o the backquotes.